src/test/java/uk/gov/dhsc/htbhf/page/PageName.java

```diff
@@ -10,36 +10,38 @@
  */
 public enum PageName {
 
+    APPLY("Apply for Healthy Start"),
     ARE_YOU_PREGNANT("are you pregnant"),
     CHECK_ANSWERS("check answers"),
     CHILD_DATE_OF_BIRTH("enter your childrens dates of birth"),
-    DECISION("decision"),
+    COOKIES("cookies"),
     DATE_OF_BIRTH("enter date of birth"),
+    DECISION("decision"),
     DO_YOU_HAVE_CHILDREN("do you have children under four years old"),
+    ELIGIBILITY("Eligibility"),
     EMAIL_ADDRESS("email address"),
     ENTER_CODE("enter code"),
-    POSTCODE("postcode", ADDRESS_LOOKUP),
-    SELECT_ADDRESS("select address", ADDRESS_LOOKUP),
+    HOW_IT_WORKS("How it works"),
+    IN_SCOTLAND("I live in Scotland"),
     MANUAL_ADDRESS("manual address"),
     NAME("enter name"),
+    TEST_NAMES("test name"),
```

Comment:
I don't think this one should be here, it should be removed.

Reply:
placed in alphabetical order